src/physics/base_solver.hpp

```diff
@@ -179,6 +179,16 @@ class BaseSolver {
    */
   std::string root_name_;
 
+  /**
+   * @brief Number of significant figures to output for floating-point
+   */
+  static constexpr int FLOAT_PRECISION_ = 8;
+
+  /**
+   * @brief Damping amount for ODE integration methods
+   */
+  static constexpr double DAMPING_ = 0.5;
```

Comment:
It looks like this might be a way of silencing a "magic number" warning, but I don't believe the 0.5 values in the related ODESolver constructors are related to damping.